### ISSUE TITLE
X-com 2 WotC and Hitman segfaults with Mangohud

### ISSUE THREAD

**Issue by Tontsu** [flightlessmango/MangoHud#285]
Mangohud crashes X-com 2 WotC and Hitman with `mangohud %command%` launch option. The crash message is identical in both games. The games are working fine without mangohud launch option. Vanilla X-com 2 and other games are working fine with Mangohud.

Crash message: 
```shell
MANGOHUD: can't find file pci.ids
Version: 4.6 
SDL2 initialised [built against 2.0.5, running with 2.0.5]
MANGOHUD: can't find file pci.ids
Version: 4.5 
MANGOHUD: can't find file pci.ids
Version: 4.5 
XCOM2WotC: crash reporter initialised with path "/home/tontsu/.local/share/feral-interactive/XCOM 2 WotC/crashes"
XCOM2WotC: dumped to "/home/tontsu/.local/share/feral-interactive/XCOM 2 WotC/crashes/471641fe-c96f-4a4c-23b45beb-4c244056.dmp"
XCOM2WotC: crash reporter "/home/tontsu/.steam/debian-installation/steamapps/common/XCOM 2/XCOM2WotC/bin/feral_linux_crash_reporter" launching
Game crashed with signal 11
Game crashed with signal SIGSEGV(11): Segmentation fault.

If possible, launch Steam from command line to check the output when the game is run.
Then, contact support@feralinteractive.com with the details of the output, your Steam System Info, as well as the dump file:
/home/tontsu/.local/share/feral-interactive/XCOM 2 WotC/crashes/471641fe-c96f-4a4c-23b45beb-4c244056.dmp

```

Link to the crash dump:

[https://github.com/Tontsu/crashes/blob/master/471641fe-c96f-4a4c-23b45beb-4c244056.dmp](url)

Specs;

Pop_os 20.04
Ryzen 3600
Geforce GTX 1070

**Comment by flightlessmango** [comment]
Does it crash the game or the launcher?

**Comment by Tontsu** [comment]
It crashes the game even if the Feral launcher is disabled.

**Comment by flightlessmango** [comment]
is this still reproducible?

**Comment by userofryzen** [comment]
yes, for me it's reproducible yet

**Comment by flightlessmango** [comment]
Tested with the demo of Hitman 2016. Feral launcher doesn't work, however while skipping the launcher mangohud works just fine

**Comment by userofryzen** [comment]
skipping the launcher is giving me the same crash ._.

**Comment by userofryzen** [comment]
how do you skip the launcher.. may be I am doing wrong. 

PD. I have game mode installed and it's probable being used by hitman

**Comment by flightlessmango** [comment]
There is a box in the launcher you can untick so it doesn't start, so just run it once without mangohud

**Comment by userofryzen** [comment]
okey, Idk why before it was not working, but here it is. 
mango is not showed.. but at least it won't crash the game

What launch options have you put in steam for hitman to be launched?
For me mango is not showed..

**Comment by flightlessmango** [comment]
I'm just using `mangohud %command%`

**Comment by userofryzen** [comment]
the demo I think is different from the "main" version. 
The demo may be it's opengl (?)

I have tried your launch options and nothing. Mango HUD is not showed


things of the output  in console:
MANGOHUD: Uploading is disabled (permit_upload = 0)
SDL2 initialised [built against 2.0.5, running with 2.0.5]

But at least it won't crash the game while skipping the launcher

**Comment by userofryzen** [comment]
I have discovered that there is a vulkan renderer of the game in alpha
state. I have been testing and with the usual environment variable for
mangohud enabled in vulkan games mango is working in hitman!
The had thing is that I have only got it in a vulkan alpha state renderer
that is not gonna be developed more(:() but it's here. I don't know why
with OpenGL 4.6 renderer is not working.

Btw I have found a command that gives me possibilities to not only enable
vulkan but to enable an overlay that gives me little info about FPS ,
renderer etc. Not as much as mango but it's usable.
